table/table.go

```diff
@@ -578,3 +570,205 @@ func (opt traceOption) ApplyTableOption(opts *Options) {
 func WithTrace(t trace.Table) traceOption { //nolint:gocritic
 	return traceOption{t: &t}
 }
+
+type (
+	BulkUpsertRequest Ydb_Table.BulkUpsertRequest
+)
+
+type BulkUpsertData interface {
+	ApplyBulkUpsertRequest(req *BulkUpsertRequest) error
+}
+
+type BulkUpsertRows struct {
+	Rows value.Value
+}
+
+func (data BulkUpsertRows) ApplyBulkUpsertRequest(req *BulkUpsertRequest) error {
+	a := allocator.New()
```

Comment:
Need to create allocator outside from Apply... for exclude situation of multiple creating allocators
Maybe in internal/table/Client.BulkUpsert ?